po/de.po

```diff
@@ -3757,48 +3749,44 @@ msgid ""
 "into\n"
 "another type."
 msgstr ""
-"Die Rust-Integer-Typen implementieren alle [`From<T>`][1] und [`Into<T>`]"
-"[2]\n"
-"Eigenschaften, um uns zwischen ihnen umwandeln zu lassen. Das `From<T>`-"
-"Merkmal hat ein einzelnes `from()`\n"
-"-Methode und ähnlich hat das `Into<T>`-Merkmal eine einzige `into()`-"
-"Methode.\n"
-"Durch die Implementierung dieser Merkmale drückt ein Typ aus, in was er "
-"umgewandelt werden kann\n"
-"ein anderer Typ."
+"Alle Rust-Integer-Typen implementieren die [`From<T>`][1]- und [`Into<T>`]"
+"[2]-Merkmale (traits), \n"
+"um zwischen ihnen umzuwandeln. Das `From<T>`-Merkmal (trait) hat eine "
+"einzige `from()`\n"
+"-Methode und ähnlicherweise hat das `Into<T>`-Merkmal auch nur eine einzige "
```

Comment:
"ähnlicherweise" sounds strange and according to Duden, it's actually not a German word. You could write
"in ähnlicher Weise", but I would simply shorten this to "ebenso" or "genauso". These latter sound more natural to me.